imblearn/under_sampling/_prototype_selection/tests/test_edited_nearest_neighbours.py

```diff
@@ -49,7 +49,7 @@ def test_enn_init():
 
 
 def test_enn_fit_resample():
-    enn = EditedNearestNeighbours()
+    enn = EditedNearestNeighbours(n_neighbors=4)
```

Comment:
updated n_neighbors to be able to use the old code with the correct behaviour of the class